src/backend/utils/compatibility_layers.ts

```diff
@@ -372,18 +374,51 @@ export async function getGamingPortingToolkitWine(): Promise<
   return gamingPortingToolkitWine
 }
 
+export type AllowedWineFlags = Pick<
+  LaunchCommand,
+  '--wine' | '--wrapper' | '--no-wine'
+>
+
+export function getWineFlags(
+  wineBin: string,
+  wineType: WineInstallation['type'],
+  wrapper: string,
+  asArray?: false
+): AllowedWineFlags
+export function getWineFlags(
+  wineBin: string,
+  wineType: WineInstallation['type'],
+  wrapper: string,
+  asArray: true
+): string[]
```

Comment:
maybe have this be 2 functions? `getWineFlags` and `getWineFlagsArray`? I saw the `true` added when calling this function and I had to look for this code

I think it will make things easier to read

Reply:
Yup, that definitely feels better, done in 292d0ecc84e15791741dcd8125b9230e138a76d3